frontend/app/views/notes/_form.html.erb

```diff
@@ -1,18 +1,26 @@
-<% 
+<%
+  # define these when nesting notes within a subrecord
+  nested_note_jsonmodel ||= nil
+  nested_in_jsonmodel ||=nil
+
```

Comment:
We've never nested notes in a nested record before and these templates needed to be tinkered to handle the deeper paths.  `nested_in_jsonmodel` is passed through to indicate we're dealing with a "nested" note.